nixos/modules/services/misc/prowlarr.nix

```diff
@@ -37,5 +60,17 @@ in
     networking.firewall = mkIf cfg.openFirewall {
       allowedTCPPorts = [ 9696 ];
     };
+
+    users.users = mkIf (cfg.user == "prowlarr") {
+      prowlarr = {
+        group = cfg.group;
+        home = cfg.dataDir;
+        uid = config.ids.uids.prowlarr;
```

Comment:
wouldn't the ID have to be defined in https://github.com/NixOS/nixpkgs/blob/master/nixos/modules/misc/ids.nix first?

Reply:
Yep, you're absolutely right! Forgot to stage the file, thank you 🙇